### PR TITLE
data:vidhimakani74

### PR DESCRIPTION
**Pull request by vidhimakani74**
portfolio website and milestone section added

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3649"></a>

### CODE REVIEW

File: data/vidhimakani74.json [hunk 1]
```diff
@@ -41,5 +54,16 @@
       "url": "https://www.hackerrank.com/vidhimakani007",
       "icon": "SiHackerrank"
     }
+  ],
+  
+  "milestones":[
+  {
+      "title": "Made Digital Token System For Jansevakendra",
+      "date": "October 2022",
+      "icon": "code",
```
Comment (Pradumnasaraf) [inline data/vidhimakani74.json:63]:
```suggestion
      "icon": "FaCode",
```

File: data/vidhimakani74.json [hunk 2]
```diff
@@ -5,17 +5,30 @@
...
     }
   ],
+  
   "avatar": "https://github.com/vidhimakani74.png",
 
   "links": [
+     {
+      "name": "Check out my Website",
+      "url": "https://vidhimakani.netlify.app/",
+      "icon": "globe"
```
Comment (Pradumnasaraf) [inline data/vidhimakani74.json:30]:
```suggestion
      "icon": "FaGlobe"
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, 

The `icon"` value is not supported. I have suggested the fix. If you still want to add something different, please check the available icons docs - https://linkfree.eddiehub.io/icons

**Comment by vidhimakani74** [comment]
> Hey,
> 
> The `"icon"` value provided by you is not supported. I have suggested the fix. If you still want to add something different, please check the available icons docs - https://linkfree.eddiehub.io/icons

How to see preview of our changes?

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good, Thanks! 🎉

**Comment by kumarsonsoff3** [comment]
> > Hey,
> > The `"icon"` value provided by you is not supported. I have suggested the fix. If you still want to add something different, please check the available icons docs - https://linkfree.eddiehub.io/icons
> 
> How to see preview of our changes?

You'll see the preview of these changes once the PR will be merged, & deployed.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by vidhimakani74** [comment]
> 

Thanks ✨  thank you for merging

**Pull request merged**